scripts/installUbuntuDeps.sh

```diff
@@ -3,9 +3,24 @@
 install_apt_deps(){
   ${SUDO} apt-get update -y
   ${SUDO} apt-get install git make gcc g++ libglib2.0-dev docbook2x pkg-config -y
-  ${SUDO} apt-get install libboost-regex-dev libboost-thread-dev libboost-system-dev liblog4cxx-dev rabbitmq-server -y
-  ${SUDO} apt-get install mongodb openjdk-8-jre curl libboost-test-dev nasm yasm gyp libx11-dev libkrb5-dev intel-gpu-tools -y
+  ${SUDO} apt-get install libboost-regex-dev libboost-thread-dev libboost-system-dev liblog4cxx-dev -y
+  ${SUDO} apt-get install openjdk-8-jre curl libboost-test-dev nasm yasm gyp libx11-dev libkrb5-dev intel-gpu-tools -y
   ${SUDO} apt-get install m4 autoconf libtool automake cmake libfreetype6-dev libgstreamer-plugins-base1.0-dev -y
+  if [ "$GITHUB_ACTIONS" != "true" ]; then
+    ${SUDO} apt-get install rabbitmq-server mongodb -y
+  else
+    if [ -d $LIB_DIR ]; then
+      echo "Installing mongodb-org from tar"
+      ${SUDO} apt-get install -y libcurl4 openssl liblzma5
+      wget -P $LIB_DIR https://fastdl.mongodb.org/linux/mongodb-linux-x86_64-ubuntu1804-4.4.6.tgz
+      tar -zxvf $LIB_DIR/mongodb-linux-x86_64-ubuntu1804-4.4.6.tgz -C $LIB_DIR
+      ${SUDO} ln -s $LIB_DIR/mongodb-linux-x86_64-ubuntu1804-4.4.6/bin/* /usr/local/bin/
+      ${SUDO} systemctl start mongod
```

Comment:
Why start mongod in installDeps step?

Reply:
I found init-mongo was deleted by `sed -i "/init-mongo/d" bin/init-all.sh` in scripts for conference CI, so I guess it should be started earlier in test environment? Maybe it should be started in init-webtransport.sh.

Reply:
Moved to init-webtransport.sh.